js/apps/admin-ui/src/groups/Members.tsx

```diff
@@ -104,8 +120,11 @@ export const Members = () => {
       max,
     });
 
-    if (includeSubGroup) {
-      const subGroups = getSubGroups(currentGroup?.subGroups || []);
+    if (includeSubGroup && currentGroup?.subGroupCount && currentGroup.id) {
+      const subGroups = await getSubGroups(
+        currentGroup.id,
+        currentGroup.subGroupCount,
+      );
       for (const group of subGroups) {
         members = members.concat(
           await adminClient.groups.listMembers({ id: group.id! }),
```

Comment:
This request is also waiting on each other request, and can likely run concurrently.